lib/cdo/code_timer.rb

```diff
@@ -0,0 +1,13 @@
+class CodeTimer
```

Comment:
Tests? And a comment explaining the purpose of the class and example usage would be nice. :)